npm/private/npm_translate_lock.bzl

```diff
@@ -146,12 +146,15 @@ _ATTRS = {
         doc = """If true, runs preinstall, install and postinstall lifecycle hooks on npm packages if they exist""",
         default = True,
     ),
+    "lifecycle_hooks_env": attr.string_dict(
+        doc = """A map of environment variables applied when running lifecycle hooks.""",
```

Comment:
same comment - "A map" isn't enough explanation for what keys/values a user should supply. Imagine what support requests we will get, and try to preempt them :)

Reply:
Got it, I have updated the doc here, thanks.